students/meet_shah/lesson02/print_grid.py

```diff
@@ -0,0 +1,54 @@
+def print_grid(x):
+    topstart = "+ "
+    topmid   = "- "
+    topend   = "+"
+
+    midstart = "| "
+    midmid   = "  "
+    midend   = "|"
+
+    repeat = int(x/2)
+    topline = topstart + topmid*repeat + topstart + topmid*repeat + topend
```

Comment:
You can multiply help make it more DRY and less typing. Eg:
```
topline = (topstart + topmid * repeat) * 2 + topend
```

Reply:
ah sweet! totally missed that.